File: src/test/java/aoc2015/day04/Day04Test.java
package aoc2015.day04;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class Day04Test {

  @Test
  void part1Test() {
    var day = new Day04();

    System.out.println(day.part1("bgvyzdsv"));
    assertEquals(609043, day.part1("abcdef"));
    assertEquals(1048970, day.part1("pqrstuv"));
  }
}

File: aoc20/src/test/java/day04/Day04Test.kt
package day04

import aoc.loadExample
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class Day04Test {
    val input: String = loadExample("day04.txt")

    @Test
    fun part1Example() {
        val subject = Day04()

        assertThat(subject.part1(input))
            .isEqualTo(2)
    }
}